subcommands/docker/cmd.go

```diff
@@ -86,9 +88,9 @@ func doDockerCreds(cmd *cobra.Command, args []string) {
 
 	var config map[string]interface{}
 	bytes, err := os.ReadFile(dockerConfigFile)
-	if os.IsNotExist(err) {
+	if errors.Is(err, fs.ErrNotExist) {
 		dockerConfig := filepath.Dir(dockerConfigFile)
-		if _, err := os.Stat(filepath.Dir(dockerConfig)); os.IsNotExist(err) {
+		if _, err := os.Stat(dockerConfig); errors.Is(err, fs.ErrNotExist) {
```

Comment:
Is this really a 2nd change? i.e. the *fix* to your problem is checking if the path is writable. If so, `dockerConfigFile` becomes empty and a required flag. And this this is modernizing our error checking?